test/framework/build_log.py

```diff
@@ -138,6 +139,7 @@ def test_easybuildlog(self):
             r"%s.test_easybuildlog \[WARNING\] :: Deprecated functionality.*anotherwarning.*" % root,
             r"%s.test_easybuildlog \[WARNING\] :: Deprecated functionality.*onemorewarning.*" % root,
             r"%s.test_easybuildlog \[WARNING\] :: Deprecated functionality.*lastwarning.*" % root,
+            r"%s.test_easybuildlog \[WARNING\] :: Deprecated functionality.*thisisnotprinted.*" % root,
```

Comment:
Are you missing the `silent` test where this shouldn't be printed?

Reply:
No, it's there already...

The test checks both the messages printed to stderr (which haven't changed), and the messages included in the log file (which gets one additional line).

The log message isn't silenced, the one that is printed to stderr is.

Reply:
Ah ok